src/execution/lazy/statements.rs

```diff
@@ -112,25 +152,32 @@ impl LazyAddGraphNodeAttribute {
     }
 
     pub(super) fn evaluate(&self, exec: &mut EvaluationContext) -> Result<(), ExecutionError> {
-        let node = self.node.evaluate_as_graph_node(exec)?;
+        let node = self
+            .node
+            .evaluate_as_graph_node(exec)
+            .with_context(|| "Evaluating target node".to_string().into())?;
```

Comment:
Adds context that helps understand which part of the statement had a problem evaluating. Before it would only say `Expected graph node got ...`, but with multiple arguments to a statement it could be unclear which argument was at fault.